services/core/shared/pouchdb.js

```diff
@@ -28,7 +29,13 @@ const connectionPool = {};
 
 const createDb = async (name, idxList = []) => {
 	logger.debug(`Creating/opening database ${name}...`);
-	const db = new PouchDB(name, { auto_compaction: true });
+
+	let db;
+	if (name.split('/')[1] === config.db.collections.delegates.name) {
+		db = new PouchDB(name, { adapter: 'memory', auto_compaction: true });
+	} else {
+		db = new PouchDB(name, { auto_compaction: true });
+	}
```

Comment:
Please make the adapter configurable in the config and keep PouchDB independent from the product.